spec/rails_helper.rb

```diff
@@ -117,6 +117,7 @@ def self.capybara_tmp_path
   config.around(:each, skip_transaction: true) do |example|
     config.use_transactional_fixtures = false
     example.run
+  ensure
```

Comment:
I suspect this is from an un-caught exception, which we can change into an expectation. I'll help find it.

Reply:
@awwaiid it seems that after removing "ensure", all tests are passing. With that, this PR is ready for review.